### DISCUSSION TITLE
Who posted last in what thread?

### DISCUSSION THREAD

**Discussion by BilldarBagdar** [markusschloesser/MackieC4_P3#25] [Ideas]
The forums here (Issues, and Discussions) don't appear to help me quickly figure out who left the last comment in what thread (before going into the thread and figuring out by scrolling down).  Do you know how to figure that out already from the "lobby views"?  

It's easy to see who started any thread and how long ago, but that is thread-root info when what is important (to me) is who last commented, thread tip information.

For Issues, at least, it occurs to me that we could abuse the "assignment" column and "assign Issue threads to ourselves" after we comment.  So, I would comment and then assign to myself, you would comment again (as desired) and then re-assign the issue to yourself.  Rinse and repeat until we close the Issue.  That way, when I go into the "Issues lobby", it would be easy to tell which threads have comments from you I haven't answered. (and the other sides, which threads have comments from me that you haven't answered, and both sides where our roles reverse and you go into the Issues lobby)

Thoughts?

**Comment by markusschloesser** [comment]
For issues you can sort by "recently updated" (https://github.com/markusschloesser/MackieC4_P3/issues?q=is:issue+is:open+sort:updated-desc), I use that and emails (for the most stuff). 
I couldn't find a similar feature for discussions unfortunately.
Up until now I was hesitant to mention (@BilldarBagdar ) you as to not spam you too much ;-), but I also noticed that you have posted in issues/discussions for which I though you would (no critique obviously just curious), for example my fix from yesterday. 
I'M fine either way with a slight tendency towards mentions as not everything is directed towards you, some stuff just might be a reminder for myself.

**Reply by BilldarBagdar** [reply]
Ok - that will help.

I suspect you are saying I've commented on things you didn't expect me to comment on.  Eh, I might surprise you the other way next time. :)  If it bothers you, and it doesn't sound like it does, we can work out a way for you to identify posts that are reminders to yourself or whatever and I can resist the temptation to comment on those.  Maybe in a larger community of contributors in a more established project things would be less cozy.

**Reply by BilldarBagdar** [reply]
also, your fix - I wanted to say congratulations, but you "closed" the issue (in triumph, I presume) and I didn't want to re-open it just to say congrats.

**Reply by markusschloesser** [reply]
> Ok - that will help.
> 
> I suspect you are saying I've commented on things you didn't expect me to comment on. Eh, I might surprise you the other way next time. :) If it bothers you, and it doesn't sound like it does, we can work out a way for you to identify posts that are reminders to yourself or whatever and I can resist the temptation to comment on those. Maybe in a larger community of contributors in a more established project things would be less cozy.

I does definitely not bother me, I was just curious :-)
And hey, as it's just the two of us, we can only make it if we try ;-) (sorry, old song reference)
Pun aside, I am grateful for everything you do post, I can only benefit from it!

**Reply by markusschloesser** [reply]
> also, your fix - I wanted to say congratulations, but you "closed" the issue (in triumph, I presume) and I didn't want to re-open it just to say congrats.

yep, 💪😊😊😊. Having param values is fantastic, even though I wish they was more room for more characters

**Reply by BilldarBagdar** [reply]
I was not a fan.  But I remember when "Just the Two of Us" was all over the airwaves as a huge hit.  Lionel Ritchie?

**Reply by markusschloesser** [reply]
> also, your fix - I wanted to say congratulations, but you "closed" the issue (in triumph, I presume) and I didn't want to re-open it just to say congrats.

btw you can still comment on closed issues without reopening them

**Reply by markusschloesser** [reply]
> I was not a fan. But I remember when "Just the Two of Us" was all over the airwaves as a huge hit. Lionel Ritchie?

Grover Washington Jr with Bill Withers in his cheesy phase, still like it, though :-)

**Comment by BilldarBagdar** [comment]
It seems like threads with "fresh comments" rise to the top of the "Discussions" board.
